app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with model: @article, local:true do |form| %>
+
+  <% if @article.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluraliz(@article.errors.count, "error")%>
```

Comment:
typo -- it must be `pluralize`. This is preventing the article from being created successfully.